data/testSSObjects.txt

```diff
@@ -0,0 +1,129 @@
+# Flag: 1
```

Comment:
These test files should live in `tests/data`, since they aren't data for the package itself, but rather for the tests.